### ISSUE TITLE
DLSS 3 support is disabled

### ISSUE THREAD

**Issue by eXt73** [ValveSoftware/Proton#6500]
# Compatibility Report
- Name of the game with compatibility issues: Witcher 3, Cyberpunk 2077, Dying Light 2, Bright Memory: Infinite
- Steam AppID of the game: 292030, 1091500, 534380, 1178830

## System Information
- GPU: RTX 4070 Ti
- Driver/LLVM version: Nvidia 525.78.01 & 525.85.05
- Kernel version: 6.1.9-ext73-101.9-ryzen-3
- system: Kubuntu 20.04 & 22.04
- Proton version: GE-Proton7-48 & Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

I do not know if the reason lies with the drivers of Nvidia, Proton or, for example, VKD3D ... that's why I report. In the games mentioned above, it is not possible to enable 'Frame Generation' and in some games also Nvidia Reflex.

## Reproduction

Launching the above-mentioned games and You see option 'Frame Generation' is darkened in the settings

**Comment by Bitwolfies** [comment]
https://github.com/jp7677/dxvk-nvapi/issues/103

TLDR; It's not implemented, and if its anything like DLSS 2, will require NV to step in and add it themselves again. On both a driver level and to the dxvk-nvapi project.

**Comment by eXt73** [comment]
Thanks for info, that means 'the ball is on the NV's side', yes ?

**Comment by Bitwolfies** [comment]
> Thanks for info, that means 'the ball is on the NV's side', yes ?

Yes, DLSS secrets are quite guarded by team green, we only even have a working 2 cause of their involvement.

**Comment by Calinou** [comment]
> and in some games also Nvidia Reflex.

Reflex support requires installing [LatencyFleX](https://github.com/ishitatsuyuki/LatencyFleX), as official NVIDIA Reflex is not supported on Linux.

**Comment by oscarbg** [comment]
@kisak-valve  Curious if new NV 530 Linux driver branch has some progress toward supporting DLSS3 frame generarion on Proton?

**Comment by Calinou** [comment]
> @kisak-valve Curious if new NV 530 Linux driver branch has some progress toward supporting DLSS3 frame generarion on Proton?

The changelog doesn't mention anything on this aspect, so probably not. I think NVIDIA will do "user-facing" communication on this if it's ever implemented, like they did when DLSS 2 support was implemented in Proton.

**Comment by luisalvarado** [comment]
6 months later and multiple advancements in technology from AMD and still Nvidia has not pronounced themselves. This coming from someone who bought a 4090. So am just seeing 6 months of AMD awesomeness and awareness meanwhile Nvidia silent.

**Comment by Calinou** [comment]
@luisalvarado Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.

**Comment by Orum** [comment]
This issue is titled "DLSS 3", but it appears to be specific to frame generation?  I'm unable to even use DLSS spatial upscaling in The Witcher 3, despite having a Lovelace card.

**Comment by Calinou** [comment]
> This issue is titled "DLSS 3", but it appears to be specific to frame generation? I'm unable to even use DLSS spatial upscaling in The Witcher 3, despite having a Lovelace card.

Which driver version are you using? I've had Witcher 3 lock me out of DLSS/Reflex on Windows once because my driver was slightly outdated.

**Comment by Orum** [comment]
Currently on 550.78-1, which is the latest (of non-beta drivers) available in my distro.

**Comment by eXt73** [comment]
Unfortunately, Nvidia still does not fully support DLSS 3.x under Linux - including the lack of Frame Generation ... and worst of all, it does not refer to the topic at all :/

https://github.com/jp7677/dxvk-nvapi/issues/103

**Comment by Orum** [comment]
I don't think that's true, as ray reconstruction (which AFAIK is only available in DLSS 3.5 and later) works fine under Portal RTX on Linux.

**Comment by eXt73** [comment]
RR works not only there, but e.g. in Cyberpunk 2077, :) Which doesn't change the fact that FG doesn't work ;)

**Comment by mineLdiver** [comment]
As I understand it, what we do have is DLSS 2 (upscaling) and 3.5 (RR), but not DLSS 3 (FG).

**Comment by Orum** [comment]
> RR works not only there, but e.g. in Cyberpunk 2077, :) Which doesn't change the fact that FG doesn't work ;)

Sure, but your earlier post states that they don't support DLSS 3.x under Linux, which isn't true, at least for spatial upscaling.  It seems specific to The Witcher 3, or at least, that's the only game I've found that says it's unsupported.

However, it's a bit of a moot point now, as I've found TW3 just freezes way too much when running under DX12.  If that gets fixed then it becomes more important to get DLSS working.

**Comment by mutnery** [comment]
For those who have the problem of DLSS not appearing you will need to copy the game to your C drive your C drive needs to be an ssd otherwise it won't work I tried it on a couple hard drives and it's the same thing you need the game installed on the C drive and the C drive needs to be an ssd

**Comment by cgiAlexis** [comment]
> For those who have the problem of DLSS not appearing you will need to copy the game to your C drive your C drive needs to be an ssd otherwise it won't work I tried it on a couple hard drives and it's the same thing you need the game installed on the C drive and the C drive needs to be an ssd

May as well delete this reply bud, with Proton every game/program thinks it's on the C drive by default. Steam Proton creates a separate instance for each game, each game thinks it's on C.
As for SSD, not sure that's relevant.

**Comment by mineLdiver** [comment]
This should probably be closed. Latest 9.0 experimental implements DLSS Frame Generation.

**Comment by Calinou** [comment]
For reference, this was implemented in https://github.com/jp7677/dxvk-nvapi/pull/213.

**Comment by kisak-valve** [comment]
Closing per the last couple comments.
